leabra7/events.py

```diff
@@ -21,24 +22,18 @@ class Cycle(Event):
     pass
 
 
-class BeginPlusPhase(Event):
-    """The event that begins the plus phase in a trial."""
-    pass
+class BeginPhase(Event):
+    """The event that ends a phase."""
 
+    def __init__(self, phase: str) -> None:
+        self.phase = phase
 
-class EndPlusPhase(Event):
-    """The event that ends the plus phase in a trial."""
-    pass
 
+class EndPhase(Event):
+    """The event that begins a phase."""
 
-class BeginMinusPhase(Event):
-    """The event that begins the minus phase in a trial."""
-    pass
-
-
-class EndMinusPhase(Event):
-    """The event that ends the minus phase in a trial."""
-    pass
+    def __init__(self, phase: str) -> None:
+        self.phase = phase
```

Comment:
Phases don't have to be strings, since the user is creating them the user can just pass in the Phase objects (see comment below). This avoids string comparisons and an extra parameter in the phase class.